### PR TITLE
DEVTOOLS-111 Ask again when a field is empty in interactive create

### PR DESCRIPTION
**Pull request by fschargorosdky**

### CODE REVIEW

File: lib/factories/createStrategyFactory.js [hunk 1]
```diff
@@ -51,12 +53,19 @@ module.exports = function (BPromise, commandPrompt, logger, messages, errors) {
               return self.getAPIName(apis);
             }
 
+            if (err instanceof errors.EmptyFieldError) {
```
Comment (juancoen) [inline lib/factories/createStrategyFactory.js]: use a single if to check for both errors. use err.getMessage() or something like this to log the error.

File: lib/factories/createStrategyFactory.js [hunk 2]
```diff
@@ -66,6 +75,11 @@ module.exports = function (BPromise, commandPrompt, logger, messages, errors) {
               return self.getAPIVersionName(apis, apiId);
             }
 
+            if (err instanceof errors.EmptyFieldError) {
```
Comment (juancoen) [inline lib/factories/createStrategyFactory.js]: same here

File: lib/factories/createStrategyFactory.js (hunk 1 above)
Comment (fschargorosdky) [inline lib/factories/createStrategyFactory.js]: done

File: lib/factories/createStrategyFactory.js (hunk 2 above)
Comment (fschargorosdky) [inline lib/factories/createStrategyFactory.js]: done

**Comment by juancoen** [comment]
looks good to me

**Pull request merged**